src/models/member/session.coffee

```diff
@@ -21,6 +21,10 @@ LYT.session = do ->
       if credentials.status is 'ok'
         log.message 'Session: init: reading credentials from getNotaAuthToken()'
         LYT.session.setCredentials credentials.username, credentials.token
+        LYT.service.logOn credentials.username, credentials.token
```

Comment:
What if this `logOn` fails?

Reply:
Should trigger an error that redirects to nota.dk